schema.sql

```diff
@@ -1,5 +1,7 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- To creat a database named as "vet_clinic"
+CREATE DATABASE vet_clinic;
+
+-- To create a table named as "animals"
+CREATE TABLE animals ( id INT, name CHAR(255), date_of_birth DATE, escape_attempts INT, neutered BOOLEAN, weight_kg DECIMAL(5,2));
```

Comment:
- [ ] [OPTIONAL] I would suggest that you make the **Id column a PRIMARY key** for the animals table 😃 👍

Reply:
Thank you so much for the kind review and close attention to the schema.
I altered the table to make set the id data type as PRIMARY KEY.